### ISSUE TITLE
Add --team to remaining fly commands

### ISSUE THREAD

**Issue by taylorsilva** [concourse/concourse#5215]
## Goal

Implement `--team` for remaining fly commands. `fly/integration` tests should be updated. Cases should be added to the `error_handling` file as well https://github.com/concourse/concourse/blob/587483424a47f3eebb2ed8e861687b60cccb9ec7/fly/integration/error_handling_test.go#L92

```bash
# use default team
fly -t ci <command> <command-options>
# use another team
fly -t ci <command> <command-options> --team other-team
```

- [ ] abort-build
- [ ] resources
- [ ] check-resource-type
- [ ] check-resource
- [ ] resource-versions
- [ ] watch
- [ ] checklist
- [ ] clear-task-cache
- [ ] execute
- [ ] expose-pipeline
- [ ] destroy-pipeline
- [ ] get-pipeline
- [ ] set-pipeline
- [ ] hide-pipeline
- [ ] pause-pipeline
- [ ] unpause-pipeline
- [ ] rename-pipeline
- [ ] order-pipelines
- [x] ~jobs~
- [x] ~trigger-job~
- [x] ~pause-job~
- [x] ~unpause-job~
- [x] ~hijack~

## Background
Follow-up story from #4196. You can look at `trigger-job` for guidance on how to implement the flag.

https://github.com/concourse/concourse/blob/587483424a47f3eebb2ed8e861687b60cccb9ec7/fly/commands/trigger_job.go#L40-L47

**Comment by jroden** [comment]
Hi @taylorsilva hope you had a good weekend!  I would like to take one of these as a first contribution to concourse :smile: Picking one at random, I'll start on set-pipeline assuming its available?

**Comment by taylorsilva** [comment]
@jroden `set-pipeline` is available to work on! Looking forward to your contribution :)

**Comment by khng** [comment]
Hi @taylorsilva! 

Was hoping to pick up the `destroy-pipeline` command for this issue. 

Let me know if there are any issues!

**Comment by taylorsilva** [comment]
All yours! Thanks for contributing :)

**Comment by techgaun** [comment]
@taylorsilva is the `get-pipeline` available to pick? I would like to take that if its free. Same goes for `expose-pipeline` but would like to tackle `get-pipeline` first.

**Comment by taylorsilva** [comment]
@techgaun they're both yours! Thanks for the help :)

**Comment by techgaun** [comment]
@taylorsilva I would like to take the `hide-pipeline` as well. Also, I mentioned about this on the PR: Would it be possible to get invited to `Contributors` team? Thank you!

**Comment by taylorsilva** [comment]
@techgaun yup, all yours! Thanks for the help!

I'm not sure how the contributors team works, pinging @xtremerui because he set it up. Do you know how to add people to the contributor team Rui? Ping me again or in your latest PR if you're still not part of it.

**Comment by xtremerui** [comment]
@techgaun in the first time you submit a PR to concourse/concourse, you should receive an invitation from Github for joining `Contributors` team. Could you check your spam/junk folder of your email? Please let me know if not. Thx!

**Comment by techgaun** [comment]
@xtremerui yep I checked my inbox including spam once I saw that on contributing guide but I never saw the e-mail anywhere yet. I saw that other contributor(s) might have faced similar issues.

**Comment by xtremerui** [comment]
@techgaun thx for the info. I think the bot we used was malfunctioned so it didn't actually send out the invitation. Now I have fixed it and resent the github event to feed the bot. You should get an invite now. Let me know. Thx.

**Comment by techgaun** [comment]
@xtremerui all set now. I got the invite. thanks for setting that up.

**Comment by techgaun** [comment]
@taylorsilva looks like https://github.com/concourse/concourse/pull/5917 added team arg for `hide-pipeline` already.

**Comment by taylorsilva** [comment]
Thanks for pointing that out! Updated the comment. Thanks again for your help!

**Comment by tech-geek29** [comment]
@taylorsilva is the destroy-pipeline available to pick? I would like to pick that if its free!

**Comment by taylorsilva** [comment]
@tech-geek29 all yours, thank you!

**Comment by tech-geek29** [comment]
@taylorsilva I think destroy-pipeline is already done & merged! I think it should be checked in above Goal.  

I can pick up checklist. I see that is free!

**Comment by taylorsilva** [comment]
It is, wonder when that got it 🤔 Feel free to do any of them and tag this issue in the PR

**Comment by tech-geek29** [comment]
@taylorsilva I do not have contributor access role. I am not able to add label to PR. What is the process for that?

**Comment by aoldershaw** [comment]
@tech-geek29 I just approved your request to join contributors in https://github.com/concourse/governance (welcome!) - can you try to add the label again?

**Comment by tech-geek29** [comment]
Thanks very much @aoldershaw , I was able to attach labels. @taylorsilva @aoldershaw I have raised PR #7162. Can you please review and approve it?

**Comment by taylorsilva** [comment]
It'll be assigned to someone tomorrow during standup for review.

**Comment by tech-geek29** [comment]
Awesome! Thanks @taylorsilva

**Comment by dhantha** [comment]
Hi @taylorsilva I would like to work on fly watch

**Comment by taylorsilva** [comment]
@dhantha sorry for the late reply, but go for it!

**Comment by Caprowni** [comment]
@taylorsilva are some of these still up for grabs? (The ones that are unticked) Will take a look at getting to a couple of them :)

**Comment by taylorsilva** [comment]
@Caprowni the list is up to date, tackle any that you like :)

**Comment by taylorsilva** [comment]
List is still up to date

**Comment by Caprowni** [comment]
Hey 👋 @taylorsilva I think there are a few that need ticked off the list that have already been done:

check-resource
check-resource-type
resources
resource-versions
archive-pipeline

**Comment by taylorsilva** [comment]
List is now _definitely_ up to date. Thanks @Caprowni!

Sorted the list too so unfinished items are at the top.

**Comment by Priyanshinv** [comment]
Hi @taylorsilva , I've raised a pull request(#8940) for adding --team to rename-pipeline command. Could you please review it?

**Comment by Priyanshinv** [comment]
Hi @taylorsilva , have raised a PR #8940 for adding --team flag to rename-pipeline command. Could you please review it?

**Comment by jesslourenco** [comment]
Hello @taylorsilva, I would like to contribute with `pause-pipeline` :) 
Also, you might want to check `rename-pipeline` in the list

**Comment by Petahhh** [comment]
I'll take `execute`

**Comment by evgnblkn** [comment]
Hi! Please add `--team` to `clear-resource-cache`

**Comment by IvanChalukov** [comment]
I've open a [PR](#9106) for adding of `--team` option to `clear-resource-cache`, but I cannot see command in list above. @taylorsilva, can I add the `--team` option to `fly containers`? I noticed it's missing from the list too. Should `fly containers `have the `--team` option, or was it intentionally excluded?

**Comment by taylorsilva** [comment]
@IvanChalukov feel free to add it! I think the list got incorrectly updated at some point so might be out of sync from reality
